### PR TITLE
Updates on Landing Zone Guide

### PR DESCRIPTION
**Pull request by marinalimeira**
This will probably conflict with #387 but the changes are smaller and we can merge soon.

I hit those while deploying the guide by myself.  I also hit [this error](https://github.com/hashicorp/terraform/issues/23615) that Rho also [mention here](https://github.com/rhoboat/ref-arch-self-deploy-docs/blob/main/1_deploy_landing_zone.md). Maybe we add a section for it?

### CODE REVIEW

**Review by ina-stoyanova — DISMISSED** [review]
Looks good.

**Review by rhoboat — DISMISSED** [review]
Thanks Marina!

File: _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc [hunk 1]
```diff
@@ -1038,7 +1038,7 @@ Next, use the `account-baseline-root` module from the Gruntwork Infrastructure a
 [source,hcl]
 ----
 module "root_baseline" {
-  source = "git::git@github.com:gruntwork-io/module-security.git//modules/account-baseline-root?ref=v0.36.0"
+  source = "git::ssh://git@github.com:gruntwork-io/terraform-aws-security.git//modules/account-baseline-root?ref=v0.44.10"
```
Comment (brikis98) [inline _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc:1041]: Did any of the input or output vars change across these 8 backwards incompatible versions?

**Review by brikis98 — COMMENTED** [review]
Thx for the fixes! It's def a good improvement, but we need to sanity check all the releases between v0.36.0 to v0.44.10 to make sure no input/output vars were removed, renamed, or new required ones added.

**Comment by josh-padnick** [comment]
All, are we ready to merge this? What remains to be done?

**Comment by josh-padnick** [comment]
Bump. @brikis98 Any thoughts on if we can merge this?

File: _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc (hunk 1 above)
Comment (brikis98) [reply _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc:1041]: Before merging, @marinalimeira could you look into this?

**Comment by marinalimeira** [comment]
Sorry for the delay. I am coming back to this one now.

File: _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc (hunk 1 above)
Comment (marinalimeira) [reply _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc]: There are some new outputs: `organization_root_id`, `aws_ebs_encryption_by_default_enabled` and `aws_ebs_encryption_default_kms_key`

Some variables were added: `iam_password_policy_allow_users_to_change_password`, `iam_password_policy_hard_expiry`, `encrypted_volumes_kms_id`, `rds_storage_encrypted_kms_id`, `iam_role_tags`, `cloudtrail_allow_kms_describe_key_to_external_aws_accounts`, `cloudtrail_tags`, `cloudtrail_is_organization_trail`, plus variables for IAM access analyzer and EBS encryption.

None of these look like a breaking change in the existing interface for this guide. What do you think?

Comment (brikis98) [reply _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc]: Are any of those new variables _required_? Or do they all have reasonable defaults?

Comment (marinalimeira) [reply _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc]: All of them have a default.

I made these updates while testing the guide by myself. After updating the version number, was not necessary to add any other variable.

Comment (brikis98) [reply _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc]: Roger, thx for checking!

**Review by brikis98 — APPROVED** [review]
LGTM

**Pull request merged**